api/logic/modplatform/modpacksch/FTBPackInstallTask.h

```diff
@@ -8,12 +8,39 @@
 
 namespace ModpacksCH {
 
+struct MULTIMC_LOGIC_EXPORT FlaggedMod {
```

Comment:
Perhaps there should be a field with information as to what it tracks and how it may endanger the user?

Reply:
There's a `flagReason` field for that, though what it should contain is not exactly defined...